backend/lcfs/web/application.py

```diff
@@ -113,6 +110,8 @@ def get_app() -> FastAPI:
     #     print("⏳ Waiting for debugger attach on port 5678...")
     #     debugpy.wait_for_client()
 
+    setup_logging(level=logging.DEBUG)
```

Comment:
suggestion: This should either use settings.log_level or we should the setting.

Reply:
Thanks for pointing this out! I'll get the issue fixed.